data/mods/ssb/moves.js

```diff
@@ -3306,31 +3306,85 @@ let BattleMovedex = {
 		type: "Fire",
 	},
 	// Teclis
-	zekken: {
+	absoluteconfiguration: {
 		accuracy: true,
 		basePower: 0,
 		category: "Status",
-		desc: "Raises the user's Attack by 2 stages and its Speed by 1 stage.",
-		shortDesc: "Raises the user's Attack by 2 and Speed by 1.",
-		id: "zekken",
-		name: "Zekken",
+		desc: "Puts the foe to sleep. Summons a Nightmare Field with the effects of Nightmare for 4 turns.",
+		shortDesc: "Puts foe to sleep. Nightmare for 4 turns.",
+		id: "absoluteconfiguration",
+		name: "Absolute Configuration",
 		isNonstandard: "Custom",
-		pp: 10,
+		pp: 1,
 		priority: 0,
-		flags: {snatch: 1, mirror: 1},
+		flags: {},
 		onTryMove() {
 			this.attrLastMove('[still]');
 		},
 		onPrepareHit(target, source) {
-			this.add('-anim', source, "Swords Dance", source);
+			this.add('-anim', source, 'Dark Pulse', target);
+			this.add('-anim', source, 'Dark Void', target);
 		},
-		boosts: {
-			atk: 2,
-			spe: 1,
+		status: 'slp',
+		isZ: "darkrainiumz",
+		secondary: {
+			chance: 100,
+			self: {
+				onHit() {
+					this.field.setTerrain('nightmarefield');
```

Comment:
Id run this in an onHit rather than as a secondary, prevents possible suppression of secondaries.